dist/pinny.js

```diff
@@ -443,6 +445,19 @@
             return $(document.activeElement);
         },
 
+        /*
+        * In iOS, there is an issue when soft keyboard triggers, position fixed
+        * elements on screen don't get updated, distorting the UI.
+        * Fix: blur keyboard on orientationchange
+        */
+        _blurOnOrientationChange: function($pinny) {
+            if ($.os.ios) {
+                $(window).on('orientationchange', function(e) {
+                    $pinny.find(':focus').blur();
```

Comment:
Right, that's why I asked if one should check if it's an input. But AFAIK you can still call blur on non-inputs and it shouldn't cause any problems, so I was asking if it would make sense to skip that check.

That makes sense. Not sure what it has to do with this orientation change. I was merely offering activeElement as an alternative, per Steve's comment. However, where is this anyway? I don't see _hideSpacer in pinny.js